front/components/assistant/conversation/content_creation/ShareContentCreationFilePopover.tsx

```diff
@@ -202,7 +207,7 @@ export function ShareContentCreationFilePopover({
                   icon={InformationCircleIcon}
                 >
                   {isSharingForbidden
-                    ? "Your workspace administrator has turned off sharing of Content Creation files."
+                    ? "Your workspace administrator has turned off sharing of Content Creation files with the public."
```

Comment:
I think this is a bit cleaner?
```suggestion
                    ? "Your workspace administrator has turned off public sharing of Content Creation files"
```